packages/p2p/src/utils/date-time.js

```diff
@@ -50,3 +50,13 @@ export const secondsToTimer = distance => {
 
     return `${toDoubleDigits(hours)}:${toDoubleDigits(minutes)}:${toDoubleDigits(seconds)}`;
 };
+
+// The function below returns the time difference in hours between the current time and the time set
```

Comment:
could you add the jsdocs for this as well since this is a new function and we have done it already for other functions during the refactor of this file 🙏